polyfills.js

```diff
@@ -14,12 +14,14 @@ try {
   process.cwd()
 } catch (er) {}
 
-var chdir = process.chdir
-process.chdir = function(d) {
-  cwd = null
-  chdir.call(process, d)
+if (process.chdir != null) {
```

Comment:
```suggestion
// This check is needed until node.js 12 is required
if (process.chdir != null) {
```

It seems what you've run into is only an issue where the `--experimental-worker` flag is required, newer versions provide a `process.chdir` function but set `.disabled = true` for worker threads.  Let's add a comment so we know why this conditional exists and when it can be removed.

Reply:
That's a jolly good idea! I had taken the Node documentation at face-value, whereby I understood "this feature is not available" as "the function is undefined". The `disabled` flag is... surprising, to say the least, but it's good it allows getting rid of cruft once node 10 drops EOL.